src/sentry/snuba/discover.py

```diff
@@ -80,11 +93,6 @@ class EventsMeta(TypedDict):
 EventsMeta.__annotations__["isMetricsExtractedData"] = bool
 
 
-class EventsResponse(TypedDict):
-    data: list[dict[str, Any]]
-    meta: EventsMeta
-
-
```

Comment:
Removed this, its duplicated in search.events.types

Reply:
turns out, shouldn't have removed this, needed for the response typing of the events endpoint, moved this there instead